src/main/java/com/mael/ttt/ui/MenuOption.java

```diff
@@ -22,14 +22,21 @@ public String getMenuOptionText() {
         return menuOptionText;
     }
 
-
-    public static MenuOption idToOption(String menuOptionId) {
+    public static MenuOption idToOption(String id) {
         for (MenuOption menuOption : values()) {
-            if (menuOptionId.equals(menuOption.getMenuOptionId())) {
+            if (id != null && id.equals(menuOption.getMenuOptionId())) {
```

Comment:
OMG SO SIMPLE
why can't I spot these things.

I agree with the naming duplication, gotta add those changes now. :+1: